Plex: migrate play media action to dedicated page (#46573)
home-assistant/home-assistant.github.io: source/_integrations/plex.markdown

**Token**: A valid authorization token for your Plex server. If provided without 'Host', a connection URL will be retrieved from Plex.

{% include integrations/actions.md %}

> 

Reviewer: According to the [integration_docs_template](https://github.com/home-assistant/home-assistant.io/tree/current/source/_integrations/_integration_docs_template.markdown), the list of actions should be after the supported entities:

Author: Done

Automatic upgrades of Plex Media Server can be triggered for some installation types, such as Windows and certain NAS devices.

{% include integrations/actions.md %}

> ## Media player

Author: ```suggestion
## Media player
```

To play Plex music directly to Sonos speakers, the following requirements must be met:

1. Remote access enabled for your Plex server.
2. Sonos speakers linked to your Plex account. For setup instructions, refer to [Control Sonos playback with a Plex app](https://support.plex.tv/articles/control-sonos-playback-with-a-plex-app/).
3. [Sonos](/integrations/sonos/) integration configured.

> Call the [Play specified media](/actions/media_player.play_media/) action with the `entity_id` of a Sonos integration device and `media_content_type` prepended with `plex://`. Both `music` and `playlist` `media_content_type` values are supported.

Reviewer: Was `media_content_type` added here by mistake?
```suggestion
Call the [Play specified media](/actions/media_player.play_media/) action with the `entity_id` of a Sonos integration device and `media_content_type` prepended with `plex://`. Both `music` and `playlist` values are supported.
```

Author: @Laufmaschine Yes, but this was poorly phrased.

```suggestion
Call the [Play specified media](/actions/media_player.play_media/) action with the `entity_id` of a Sonos integration device and `media_content_type` prepended with `plex://`. Valid `media_content_type` values are `music` and `playlist`.
```
